flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTestUtil.java

```diff
@@ -40,6 +41,25 @@
  * TODO: Move util methods to CatalogTest and remove this class
  */
 public class CatalogTestUtil {
+	public static void checkEquals(CatalogTable t1, CatalogTable t2) {
```

Comment:
Since CatalogTable has multiple of subclasses, we probably need to make sure that the classes are equal too.